js_modules/dagit/packages/core/src/runs/RunActionButtons.tsx

```diff
@@ -286,35 +286,6 @@ function usePipelineAvailabilityErrorForRun(
       };
     }
 
-    if (matchType === 'snapshot-only') {
```

Comment:
The `matchType === 'origin-and-snapshot'` "best case" should now be used for the `'origin'` match type.